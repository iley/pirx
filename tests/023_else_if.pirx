extern func main(): int {
  var x: int = 15;
  if x < 10 {
    printf("less than 10\n");
  } else if x < 20 {
    printf("between 10 and 20\n");
  } else {
    printf("over 20\n");
  }
  return 0;
}
