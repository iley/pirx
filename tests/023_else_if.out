between 10 and 20
